### DISCUSSION TITLE
First testcase taking too long to execute.

### DISCUSSION THREAD

**Discussion by ayush933** [cpeditor/cpeditor#988] [Q&A] [answered]
I recently moved to a new laptop and imported all my cpeditor settings from the old laptop.
On the old laptop cpeditor was working fine but on the new one, the first testcase of each program takes significantly longer to execute.

![image](https://user-images.githubusercontent.com/21211904/132106002-9d5540c3-b460-4ef9-8fee-dd3ebab7362b.png)

This is the result of 2 exact same test cases. 
It even happens on empty programs and sometimes even times out making the editor almost unusable.

**Comment by ouuan** [comment, marked as answer]
https://cpeditor.org/docs/faq/#my-code-gets-tle-on-every-first-execution-after-compilation
